src/Sylius/Behat/Context/Api/Shop/CheckoutContext.php

```diff
@@ -300,6 +300,7 @@ public function iProvideAdditionalNotesLike(string $notes): void
 
     /**
      * @Given I confirmed my order
+     * @Given I have confirmed order
```

Comment:
I would prefer to do order management based on commands if possible instead of API calls